src/lib.rs

```diff
@@ -244,6 +246,34 @@ fn strip_prefix() -> impl tera::Function {
     )
 }
 
```

Comment:
Even the links are clickable, they are not differentiable from normal text. Adding `link` class should do the job

Reply:
Good point. I've added it. As `link` seems to be used only dynamically it didn't worked at the beginning and therefore i've put it now to the `safelist` of `tailwind`.